rfcs/experimental/kernel_templates/kernel_configuration/README.md

```diff
@@ -0,0 +1,45 @@
+# Kernel Configuration
+
+## Introduction
+
+`kernel_param` is a generic structure for configuring SYCL kernels.
+Each kernel template is invoked with a `kernel_param` type or object.
+Typically, a kernel template supports multiple values for the configuration parameters.
+Optimal values may depend on the invoked kernel, the data size and type(s),
+as well as on the used device.
+
+## Signature
+
+```c++
+// defined in <oneapi/dpl/experimental/kernel_templates>
+// in namespace oneapi::dpl::experimental::kt
+
+template <std::uint16_t DataPerWorkItem,
+          std::uint16_t WorkGroupSize,
+          typename KernelName = /*unspecified*/>
+struct kernel_param;
+```
+
+The library guide describes all the signature details in
+https://github.com/uxlfoundation/oneDPL/blob/oneDPL-2022.8.0-release/documentation/library_guide/kernel_templates/kernel_configuration.rst.
+
+## Open Questions
+
+### Runtime Parameters
+
+The `data_per_workitem` and `workgroup_size` parameters are currently provided at compile time,
+but their optimal values depend on the number of elements to process, which is known only at run time.
+
+It should be investigated whether these parameters can be made run-time configurable
+without compromising efficiency.
```

Comment:
It sounds like an "either/or" choice between compile-time and run-time parameters, but it does not have to be such.
```suggestion
It should be investigated whether dynamic, run-time configurable parameters can be supported
without compromising efficiency.
```